docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,69 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each participant or subject: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon participant registration. The CandID is not configurable.
+- **PSCID** (Project Study Center ID) is a unique configurable ID. It can be manually entered when registering a participant and may contain the site abbreviation followed by sequential or randomized characters (e.g. 'MTL0006')
+
+The format and the generation of the **PSCID** must be configured by an admin at the setup stage.
+
+## Configuration
+
+### Front End
+_not yet available_
+
+### SQL
+_not yet available_
+
+### Adding identifiers from the config.xml file
+
+Both the format and the generation of PSCIDs can be configured by the project in the `config.xml` file of the `%LORIS_ROOT%/project/` directory.
+
+PSCIDs can be created for new subjects in one of 3 ways: *sequentially generated*, *manually entered*, or *randomly generated*.
+
+1. ***sequential*** generates PSCIDs sequentially for each new participant registered. **(default)**
+
+ ```xml
+ <PSCID>
+     <generation>sequential</generation> 
+     <structure>
+         <seq type="siteAbbrev"/>
+         <seq type="numeric" minLength="4"/>
+     </structure>
+ </PSCID>
+ ```
+
+2. ***manual*** asks the user to enter the PSCID when registering a new participant.
+
+ ```xml
+ <PSCID> 
+     <generation>user</generation> 
+     <structure>
+         <seq type="alphanumeric" minLength="4"/> <!-- Ex: AAA1-->
+     </structure>
+ </PSCID>
+ ```
+
+3. ***random*** generates PSCIDs with a random numerical value for each new participant registered.
+
+ ```xml
+ <PSCID>
+     <generation>random</generation> 
+     <structure>
+         <seq type="static">PROJ</seq>
+         <seq type="siteAbbrev"/>
+         <seq type="numeric" minLength="4"/>
+     </structure>
+ </PSCID>
+ ```
+ 
+ Options for the `type` element of the `<seq>` tag are:
+  - `siteAbbrev`: A string value that will be used as a dynamic prefix. Value drawn from the `Alias` field of the `psc` table in the database.
+  - `static`: A string value that will be used as a fixed prefix. Value defined in the `config.xml` file.
```

Comment:
```suggestion
  - `static`: A string value that will be used as a fixed value. Value defined in the `config.xml` file.
```

AFAIK it doesn't need to be a prefix

Reply:
did you check the code or are you assuming ?

Reply:
https://github.com/aces/Loris/blob/major/php/libraries/SiteIDGenerator.php#L200

Reply:
I'm just assuming because it would be stupid to have a system to declare the structure of a string, if all the individual components only worked as prefixes.

Reply:
Yeah in effect this ends up always being a prefix. This is how you get the "DCC" for an ID like `DCC0123`.